CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ now have load balancers configured for them that expose those services outside
 of the Kubernetes cluster. This makes it easier to access their APIs. When
 deployed on Google Cloud, these will be
 [Internal Load Balancers](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing).
+Note that this change cannot be applied to an existing deployment; delete the
```

Comment:
Maybe embolden **cannot be applied to an existing deployment** ?

Reply:
Done.